core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -417,6 +417,22 @@ class EntityService(protected val ctx: RawlsRequestContext,
         )
     }
 
+  def cloneEntities(sourceWorkspaceContext: Workspace,
+                    destWorkspaceContext: Workspace,
+                    parentContext: RawlsRequestContext
+  ): DBIOAction[Unit, NoStream, Effect with Effect.Write] =
+    for {
+      entityProvider <- DBIO.from(getProviderWithTracing(sourceWorkspaceContext, parentContext))
+      res <- traceDBIOWithParent("clone entities", ctx) { s =>
```

Comment:
nit: use `parentContext` here instead of `ctx` to more accurately assign the sub-span to its parent